free-programming-books-ta.md

```diff
@@ -43,3 +45,13 @@
 ### PHP
 
 * [எளிய தமிழில் PHP](https://freetamilebooks.com/ebooks/learn-php-in-tamil/)
+
+
+### Ruby
+
+* [எளிய தமிழில் Ruby](http://www.kaniyam.com/download/learn-ruby-in-tamil.pdf)
+
+
+### Selenium
+
+* [எளிய தமிழில் Selenium](http://www.kaniyam.com/download/learn-selenium-in-tamil.pdf)
```

Comment:
```suggestion
* [எளிய தமிழில் Selenium](http://www.kaniyam.com/download/learn-selenium-in-tamil.pdf) (PDF)
```

Similar situation here. 

Correcting these should allow the Travis CI build to pass.